src/core/interpreter_test.cc

```diff
@@ -321,4 +321,9 @@ TEST_F(InterpreterTest, Modules) {
   EXPECT_EQ("str(\x1\x2test)", ser_.res);
 }
 
+TEST_F(InterpreterTest, OutdatedGlobals) {
+  // Check table.unpack is accessable as a global.
+  EXPECT_TRUE(Execute("return unpack{1,2,3}"));
```

Comment:
> can you add what EXPECT_EQ(...,ser_.res) too?

I added a simple one, no reason to do functional testing because we need just to export a global alias